app/models/api/area.rb

```diff
@@ -43,6 +43,10 @@ def self.all_by_area_code
     end
   end
 
+  def self.derived_datasets
+    all_by_area_code.sort_by { |area_code, _| area_code }.map { |_, area| area }.find_all { |area| area.is_derived? }
```

Comment:
For an `Array` it is 👼  but for a `Hash` it isn't as explained [here](http://stackoverflow.com/questions/20999192/is-find-all-and-select-the-same-thing).